src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx

```diff
@@ -49,16 +49,7 @@ function BottomTabBar({isLoadingApp = false}: PurposeForUsingExpensifyModalProps
             return;
         }
 
-        Welcome.isOnboardingFlowCompleted({
-            onNotCompleted: () =>
-                Navigation.navigate(
-                    // Uncomment once Stage 1 Onboarding Flow is ready
-                    //
-                    // ROUTES.ONBOARDING_PERSONAL_DETAILS
-                    //
-                    ROUTES.ONBOARD,
-                ),
-        });
+        Welcome.isOnboardingFlowCompleted({onNotCompleted: () => Navigation.navigate(ROUTES.ONBOARDING_ROOT)});
```

Comment:
Now with Stage 2 we have to go to purpose page first
```suggestion
        Welcome.isOnboardingFlowCompleted({onNotCompleted: () => Navigation.navigate(ROUTES.ONBOARDING_PURPOSE)});
```

Reply:
Yes, and Purpose is set as ROOT so it is all good in this way too